preview/Dockerfile

```diff
@@ -7,16 +7,12 @@ FROM buildpack-deps:jessie-scm
 ENV LTTNG_UST_REGISTER_TIMEOUT 0
 
 # Install .NET CLI dependencies
-RUN echo "deb [arch=amd64] http://llvm.org/apt/jessie/ llvm-toolchain-jessie-3.6 main" > /etc/apt/sources.list.d/llvm.list \
-    && wget -q -O - http://llvm.org/apt/llvm-snapshot.gpg.key|apt-key add - \
-    && apt-get update \
+RUN apt-get update \
     && apt-get install -y --no-install-recommends \
-        clang-3.5 \
         libc6 \
         libcurl3 \
         libgcc1 \
```

Comment:
This list of dependencies doesn't quite match up with the one provided in https://github.com/dotnet/cli/issues/3088#issuecomment-219855194.  Specifically, it looks like `libgssapi-krb5-2` is in that list but not in this Dockerfile, whereas `libtinfo5` is in this Dockerfile but not in that list.  Is that expected?